benchmarks/README.md

```diff
@@ -7,6 +7,18 @@ protobuf language runtime.
 
 The schema for the datasets is described in `benchmarks.proto`.
 
+The benchmark is based on some submodules. To initialize the submodues:
+
+For java:
+```
+$ ./initialize_submodule.sh java
```

Comment:
Remove this shell script and just put the build instructions here (starting from git clone)

Reply:
Done